README.md

```diff
@@ -3,10 +3,17 @@ Helm chart for deploying [dani-garcia/bitwarden_rs](https://github.com/dani-garc
 
 ## TL;DR;
 
+```console
+$ helm repo add bitwarden https://cdwv.github.io/bitwarden-k8s/
```

Comment:
You might want to look into not calling it bitwarden, to prevent issues with possible future (semi-)official helm charts from either bitwarden or bitwarden_rs

Reply:
You're right in that we should state that clearly in the readme, that the project is unofficial, even more so than `bitwarden_rs` as we aren't even connected with `dani-garcia` - we're just users who wanted to put a bitwarden server on our kubernetes cluster ;) This might be a good thing to add in a separate PR.

When it comes to naming repo alias on your pc, I went for easiest understandable option. It could be anything really: bitwarden-k8s, bitwarden-unofficial etc.